eng/jacoco-test-coverage/pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-ai-textanalytics</artifactId>
-      <version>1.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-ai-textanalytics;current} -->
+      <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-ai-textanalytics;current} -->
```

Comment:
not sure how this works, but making sure it doesn't depend on the published version

Reply:
It is current version. Every time we published a new version. There is an automatic PR created to increase the next version number of SDK. It doesn't depend on the published version.